e2e-test/src/test/java/io/cdap/plugin/sendEmail/stepsdesign/SendEmail.java

```diff
@@ -0,0 +1,92 @@
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.plugin.sendEmail.stepsdesign;
+import io.cdap.e2e.pages.actions.CdfPluginPropertiesActions;
+import io.cdap.e2e.utils.CdfHelper;
+import io.cdap.plugin.sendEmail.actions.SendEmailActions;
+import io.cdap.plugin.utils.GmailClient;
+import io.cucumber.java.en.And;
+import io.cucumber.java.en.Then;
+import org.junit.Assert;
+import stepsdesign.BeforeActions;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.HashMap;
+import javax.mail.MessagingException;
+
+/**
+ * Send-email alert plugin related step design.
+ */
+
+public class SendEmail  {
+
+  @And("Select send email alert plugin")
+  public void selectSendEmailAlert() {
+    SendEmailActions.clickOnSendEmail();
+  }
+
+  @And("And Enter send email plugin property email recipient with value: {string}")
+  public void enterInputPluginPropertyWithValue(String value) {
+    SendEmailActions.enterRecipientDetails(value);
+  }
+
+  /*@And("Click on the next button")
+  public void clickOnTheNextButton() {
+    SendEmailActions.clickOnNextButtonInAlerts();
+  }*/
+
+//  @And("Click on the confirm button")
+//  public void clickOnTheConfirmButton() {
+//    SendEmailActions.clickOnConfirmButtonInAlerts();
+//  }
+
+  @And("Click Pipeline Alert Save Button")
+  public void saveTheAlert() {
+    SendEmailActions.clickPipelineAlertSaveButton();
+  }
+
+  @Then("Verify email is received by the recipients: {string} with the subject: {string}")
+  public void verifyEmailIsReceivedByTheRecipientsWithTheSubject(String recipients, String subject) throws
+    MessagingException, IOException {
+    String[] recipientList = recipients.split(",");
+    for (String recipient : recipientList) {
+      ArrayList<HashMap<String, String>> messageList =
+        GmailClient.getGmailMessages(recipient, System.getenv("EMAIL_PASSWORD"), "INBOX");
+      boolean emailReceivedFlag = false;
+      for (HashMap<String, String> message : messageList) {
+        if (message.get("subject").equalsIgnoreCase(subject)) {
+          BeforeActions.scenario.write("Email received by the recipient: " + recipient);
+          emailReceivedFlag = true;
+          break;
+        }
+      }
+      if (!emailReceivedFlag) {
+        Assert.fail("Email not received by the recipient: " + recipient);
+      }
+    }
+  }
+
+  @And("Validate send email plugin properties")
+  public void validateStringPluginPropertiesForAlerts() {
+    CdfPluginPropertiesActions.clickValidateButton();
+    SendEmailActions.verifyIfSendEmailPluginIsValidatedSuccessfully();
+  }
+
+  @And("Verify email sent message in logs")
+  public void verifyEmailSentMessageInLogs() {
+  }
```

Comment:
Add similar comment here which is added in feature file step - (with bugId implement later)